### DISCUSSION TITLE
3 axis?

### DISCUSSION THREAD

**Discussion by enddDD** [fra589/grbl-Mega-5X#295] [Q&A] [answered]
Hi, does this work with only 3 axis or do i need to have more? Also it can clone the y axis? I want to build a router. Thanks.

**Comment by RaphaelDives** [comment, marked as answer]
Hey @enddDD
Yes it is exclusively for 5 axis routers.

No, that's a Joke! :smile:
Of course you can use it for three axis! The standard configuration on Github is for three axis, also.
(three axis is the minimum btw)

And Yes You can clone any axis to Your convenience (up to 5 in total). Just look at the config.h or the Quick start guide:
https://github.com/fra589/grbl-Mega-5X/wiki/Quick-Guide-to-Setting-Up-Your-Machine-for-the-First-Time

Cheers
Raphael

**Reply by enddDD** [reply]
> Hey @enddDD Yes it is exclusively for 5 axis routers.
> 
> No, that's a Joke! 😄 Of course you can use it for three axis! The standard configuration on Github is for three axis, also. (three axis is the minimum btw)
> 
> And Yes You can clone any axis to Your convenience (up to 5 in total). Just look at the config.h or the Quick start guide: https://github.com/fra589/grbl-Mega-5X/wiki/Quick-Guide-to-Setting-Up-Your-Machine-for-the-First-Time
> 
> Cheers Raphael

Wow, thank you for your fast reply and for informations.
